schemes/everforest/everforest.nix

```
@@ -0,0 +1,24 @@
+{
+  slug = "everforest";
+  name = "EverForest";
+  author = "sainnhe";
+  colors = {
+    base00 = "#2b3339";
+    base01 = "#323c41";
+    base02 = "#3a4248";
+    base03 = "#868d80";
+    base04 = "#d3c6aa";
+    base05 = "#d3c6aa";
+    base06 = "#e9e8d2";
+    base07 = "#fff9e8";
+    base0D = "#7fbbb3";
+    base0E = "#d699b6";
+    base0C = "#83c092";
+    base0A = "#dbbc7f";
+    base09 = "#e69875";
+    base0B = "#a7c080";
+    base08 = "#e67e80";
+    base0F = "#d699b6";
```

For now, these should not include `#`
```suggestion
    base00 = "2b3339";
    base01 = "323c41";
    base02 = "3a4248";
    base03 = "868d80";
    base04 = "d3c6aa";
    base05 = "d3c6aa";
    base06 = "e9e8d2";
    base07 = "fff9e8";
    base0D = "7fbbb3";
    base0E = "d699b6";
    base0C = "83c092";
    base0A = "dbbc7f";
    base09 = "e69875";
    base0B = "a7c080";
    base08 = "e67e80";
    base0F = "d699b6";
```